src/sass/modules/_m-schemaform.scss

```diff
@@ -17,6 +17,10 @@ legend {
   }
 }
 
+input[name="root_benefit"] { 
+  width: 4rem;
+}
```

Comment:
This was necessary to enable the cursor to more conveniently access the AdditionalInfo button.